internal/handler/handler.go

```diff
@@ -70,13 +72,30 @@ func (h *Hand) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		h.UpdateEmployee(w, r)
 		return
 	default:
-		fmt.Println("errors: not found")
 		http.NotFound(w, r)
 		return
 	}
 }
 
+func (h *Hand) getInfo(r *http.Request) error {
+	ctx := r.Context()
+	corelationIDRaw := ctx.Value(middleware.CorrelationID)
+	corelationID, ok := corelationIDRaw.(string)
+	if !ok {
+		return errors.StatusInternalServerError()
+	}
+	h.lg.WithFields(logrus.Fields{
+		"method":        r.Method,
+		"corelation_id": corelationID,
+	}).Info()
+	return nil
+}
+
 func (h *Hand) GetPositions(w http.ResponseWriter, r *http.Request) {
+	err := h.getInfo(r)
```

Comment:
As we discussed before put logger call with correlationID in Service layer (not in the handler as you did)